source/tutorials/qiime2-for-experienced-microbiome-researchers.rst

```diff
@@ -0,0 +1,296 @@
+
+closed reference clustering
+'''''''''''''''''''''''''''
+
+Closed reference clustering groups sequences together which match the same reference sequence in a database with a certain similarity.
+
+VSEARCH can do closed reference clustering with the :doc:`cluster-features-closed-reference <../plugins/available/vsearch/cluster-features-closed-reference/>` method.
+This method wraps the ``--usearch_global`` VSEARCH function.
+You can decide which reference database to cluster against with the ``--i-reference-sequences`` flag.
+The input file to this flag should be a ``.qza`` file containing a fasta file with the sequences to use as references, with QIIME 2 data type ``FeatureData[Sequence]``.
+Most people use GreenGenes or SILVA, but others curate their own databases or use other standard references (e.g. UNITE for ITS data).
+You can download the references from the links on the :ref:`QIIME 2 data resources page <marker gene db>`.
+You'll need to unzip/untar and import them as ``FeatureData[Sequence]`` artifacts, since they're provided as raw data files.
+
+Assigning taxonomy
+~~~~~~~~~~~~~~~~~~
+
+Assigning taxonomy to ASV or OTU representative sequences is covered in the :ref:`taxonomy classification
+tutorial <Taxonomy>`.
+All taxonomy assignment methods are in the :doc:`feature-classifier plugin <../plugins/available/feature-classifier/index>`.
+
+There are two main approaches for assigning taxonomy, each with multiple methods available.
+
+The first involves aligning reads to reference databases directly:
+
+- :doc:`classify-consensus-blast <../plugins/available/feature-classifier/classify-consensus-blast/>`: BLAST+ local alignment
+- :doc:`classify-consensus-vsearch <../plugins/available/feature-classifier/classify-consensus-vsearch/>`:  VSEARCH global alignment
+
+Both use the *consensus* approach of taxonomy assignment, which you can learn more about in the overview and tweak with the ``maxaccepts``, ``perc-identity``, and ``min-consensus`` parameters.
+
+The second approach uses machine learning classifiers to assign likely taxonomies to reads:
+
+- :doc:`fit-classifier-sklearn <../plugins/available/feature-classifier/fit-classifier-sklearn/>`
+- :doc:`fit-classifier-naive-bayes <../plugins/available/feature-classifier/fit-classifier-naive-bayes/>`
+
+These two functions differ in the type of machine learning model that they use.
+(To get more details on the specific models implemented, check out `the plugin's associated paper <https://doi.org/10.1186/s40168-018-0470-z>`__.)
+These methods need a pre-trained model as one of the inputs: you can either download one of the pre-trained taxonomy classifiers from the :doc:`data resources page <../data-resources/>`, or train one yourself (following the steps outlined in the :doc:`feature classifier tutorial <feature-classifier>`).
```

Comment:
those methods fit the models... then ``classify-sklearn`` is used to classify with any fitted model.

So maybe just point to the ``classify-sklearn`` method above instead of modifying the text here (sure, ``fit-classifier-sklearn`` is there for anyone to use but requires some undocumented parameter specifications to really have any fun)